kg_rag/utility.py

```diff
@@ -392,5 +392,6 @@ def interactive(question, vectorstore, node_context_df, embedding_function_for_c
         output = llm_chain.run(context=node_context_extracted, question=question)
     elif "gpt" in llm_type:
         enriched_prompt = "Context: "+ node_context_extracted + "\n" + "Question: " + question
-        output = get_GPT_response(enriched_prompt, system_prompt, llm_type, llm_type, temperature=config_data["LLM_TEMPERATURE"])
+        chat_model_id, chat_deployment_id = get_gpt35()
```

Comment:
One problem with the new line 395 is that it will always call gpt-3.5, regardless of whether the user specified other gpt models, such as gpt-4.

I think the better option here is to change line 395:
from:
chat_model_id, chat_deployment_id = get_gpt35()
to:
chat_deployment_id = chat_model_id if openai.api_type == "azure" else None

Do you agree?